### PR TITLE
Improve ensurepip's --help

### PR DESCRIPTION
**Pull request by mineo**
Commit 1:

Add a space to ensurepip's --altinstall option

This changes the output from:
```
↪ python -m ensurepip --help
[...]
  --altinstall   Make an alternate install, installing only the X.Y
                 versionedscripts (Default: pipX, pipX.Y, easy_install-X.Y)
```
to:

```
↪ ./python -m ensurepip --help
[...]
  --altinstall   Make an alternate install, installing only the X.Y versioned
                 scripts (Default: pipX, pipX.Y, easy_install-X.Y)
```

Commit 2:

Add periods to the arguments of ensurepip that didn't have it

This makes --help for all optional arguments consistent and also makes it
consistent with pip --help.

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately we couldn't find an account corresponding to your GitHub username on [bugs.python.org](https://bugs.python.org/) (b.p.o) to verify you have signed the CLA (this might be simply due to a missing "GitHub Name" entry in your b.p.o account settings). This is necessary for legal reasons before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


Thanks again to your contribution and we look forward to looking at it!

**Review by grimreaper — APPROVED** [review]

**Comment by encukou** [comment]
Thanks for doing that!
I think this doesn't need an issue/NEWS, and should to be backported to all maintained versions.
I'm still new to this, so I'll ask a more experienced core dev to check those labels before merging, in case I missed anything.

**Pull request merged**
